src/ProjectOrigin.Vault/Models/WalletSlice.cs

```diff
@@ -18,5 +18,5 @@ public enum WalletSliceState
     Registering = 3,
     Sliced = 4,
     Claimed = 7,
-    Reserved = 10,
+    Reserved = 10
```

Comment:
I thought Withdrawn was part of this status?

Reply:
You could do that but since it's the entire certificate that is being withdrawn I chose to put it on the certificate